src/include/utils/workfile_mgr.h

```diff
@@ -104,6 +104,9 @@ typedef struct workfile_set
 
 	/* Number of compressed work files */
 	uint32		num_files_compressed;
+
+	/* PID of the process creating the workfile set */
+	int			pid;
```

Comment:
Why didn't the ABI tests break after such a change in the public structure?

Reply:
This struct is only used through a pointer publicly, so this change only increased its size (which is allowed). The struct is used inline in workfile_mgr.c, but that is private